10_date_timestamps_and_lies.md

```diff
@@ -145,31 +143,10 @@ df.select(to_date($"lastUpdate").as("lastUpdate"))
 // +----------+-----+
 ```
 
-and
-
-```scala
-df.select(to_date($"lastUpdate").as("lastUpdate"))
-   .groupBy($"lastUpdate").count
-   .orderBy($"lastUpdate").printSchema
-```
-
-gives the following:
-
-```scala
-// root
-//  |-- lastUpdate: date (nullable = false)
-//  |-- count: long (nullable = false)
-```
-
 whereas,
 
 ```scala
-df.select(to_date($"lastUpdate").as("lastUpdate"))
-  .groupBy($"lastUpdate")
-  .count
-  .orderBy($"lastUpdate")
-  .filter($"lastUpdate".isNull)
-  .show
+df.filter($"lastUpdate".isNull).show
```

Comment:
I think it is better to use example which directly shows the problem. Alternatively we could use `count` - `df.filter($"lastUpdate".isNull).count.show`.

With all these `groupBy`, `orderBy` and type casting it is not clear what exactly is the point here.